Migrations/ChefsterDbContextModelSnapshot.cs

```diff
@@ -49,40 +77,50 @@ protected override void BuildModel(ModelBuilder modelBuilder)
             modelBuilder.Entity("Chefster.Models.FamilyModel", b =>
                 {
                     b.Property<string>("Id")
-                        .HasColumnType("varchar(255)");
+                        .HasColumnType("TEXT");
```

Comment:
Its switched it over to sqlite it looks like. Would it be worth keeping a sqlite migration around so we can use it for the development DB or should we get rid of them entirely?